### DISCUSSION TITLE
new issue regarding sudo command to reconnect wifi client (#1605)

### DISCUSSION THREAD

**Discussion by PapaSchIumpf** [RaspAP/raspap-webgui#1658] [Q&A]
hi everyone,

a few months ago i asked for help regarding connecting to a wifi client with command and got a solution for that.

this is the code needed for that:
sudo wpa_cli -i wlan1 select_network 0

that works fine, but only if you open the wifi client tab in the raspap UI first.

so if the raspberry is restarting and you try to connect to a network with the command, it fails.

if you do the same, but open  open the wifi client tab in the raspap UI before, it works.

i didnt noticed this before.

what could be the reason and how can i solve this?

thank you.

**Comment by billz** [comment]
The `0` value represents the network ID assigned by wpa_supplicant. When you add a new network (either via RaspAP's WiFi client UI or by editing the config directly) wpa_supplicant assigns a unique ID to it. You can view your list of networks and their corresponding IDs by executing `sudo wpa_cli -i wlan1 list_networks`. You may need to change this value if more than one network is defined.

For an alternate auto-connect method, see https://github.com/RaspAP/raspap-webgui/discussions/1563

**Comment by Rohan726** [comment]
See how I solved it: https://github.com/RaspAP/raspap-webgui/discussions/1817
